doc/src/metrics.md

```diff
@@ -0,0 +1,3 @@
+# Metrics
```

Comment:
Can the metrics be under tools instead? We already have dedicated space for each tool. For example, we could add a metrics under this section: https://model-checking.github.io/verify-rust-std/tools/kani.html